### DISCUSSION TITLE
Slow As Molasses

### DISCUSSION THREAD

**Discussion by EdGadz** [enzo1982/freac#310] [Help & Support]
Installed on Windows 11.  Set the output filter to filename after the app for some unknown reason created a folder for each song file with the default setting.  Now it's slow as molasses when multiple songs are selected.  If I select only 1 song, a 4 minute song converts in 15 seconds or so.  If I select 2 or 3 or more, it shows several minutes in the Time Left counters and each song takes several minutes to convert.  Version 1.1.6 for 64 bit windows downloaded today.

**Comment by enzo1982** [comment]
This sounds like fre:ac is I/O limited. Are your files on an external hard-drive or USB thumb drive? In that case fre:ac's parallel processing can lead to saturation of the disk's I/O and slow processing, because the program cannot read from the drive fast enough. This shouldn't happen in most cases with modern external HDDs, but it also depends on the USB controller, drivers etc.

With thumb drives it's most likely the drive itself that has limited I/O capacity, but if you are using an external HDD and your system has lots of USB ports, try connecting the drive to another one. Often some ports are attached directly to the Intel/AMD system chipset while others use 3rd party controllers (ASMedia etc.) that can be slower and/or come with inferior drivers.

Also, if you are using any tuning tools (with names like _PC Tuner_, _Registry Tweaker_, _Windows Tuner_ etc.) or used such tools in the past, those can mess up the Windows drive caching settings and lead to slow I/O in programs doing lots of random disk accesses like fre:ac. Try resetting everything to the system default settings in this case.

If this doesn't help, you can disable parallel processing in fre:ac's settings. At least processing multiple files in one go should not be slower than processing the files individually then.

Even the 15 seconds for a 4 minute song seem oddly slow, though, for any computer that can run Windows 11. Unless you are using some extreme compression settings at least. For comparison, I just tried on a 2013 dual core laptop and it converts a 4 minute song from FLAC to MP3 in 4 seconds at default settings and just under 7 seconds with SuperFast mode disabled. My 6 year old 8-core desktop does the same in 0.5 seconds at default settings and 2.6 seconds without SuperFast mode. And both systems are quite slow by today's standards.
